app/models/DisplayClient.scala

```diff
@@ -17,19 +17,30 @@
 package models
 
 import play.api.libs.json.Json
-import uk.gov.hmrc.agentmtdidentifiers.model.Client
+import uk.gov.hmrc.agentmtdidentifiers.model.{Client, Enrolment, Identifier}
 
-case class DisplayClient(hmrcRef: String, name: String, taxService: String, selected: Boolean = false)
+case class DisplayClient(
+                          hmrcRef: String,
+                          name: String,
+                          taxService: String,
+                          identifierKey: String,
+                          selected: Boolean = false
+                        )
 
 
-case object DisplayClient{
+case object DisplayClient {
 
   implicit val format = Json.format[DisplayClient]
 
   def fromClient(client: Client, selected: Boolean = false): DisplayClient = {
     val keyElements = client.enrolmentKey.split('~')
     val hmrcRef = keyElements.last
```

Comment:
What's the difference between the hmrcRef & the identifier key in this case?

I get that it's down to the different parts of the enrolment key - but if one is the identifier... what's the other one? 😅

Reply:
So hmrcRef is the actual value - identifier key is just the type eg. VRN